examples/Transfer_Learning/basic_tl.py

```diff
@@ -0,0 +1,176 @@
+## Transfer Learning
+
+# This example demonstrates BayBE's
+# {doc}`Transfer Learning </userguide/transfer_learning>` capabilities:
+# * We construct a campaign,
+# * give it access to data from a related but different task,
+# * and show how this additional information boosts performance.
+
+### Imports
```

Comment:
Until now, this part is always called "Necessary imports". So we are desyncing here. Is that intended?

Reply:
Upfront: Let's not bother with "... but the other examples use" because the examples need an overhaul altogether and we should rather try to align with this new on a form of "blueprint" (i.e. not in terms of copy and paste but how it's organized and the underlying pipeline like hashed images etc) that we can reuse.

The reason why I shortened it: in my opinion, most of the text in the old examples and this new one is too verbose / redundant / boilerplate and I would ideally like to have something that is crisp and easy to maintain. Especially headings should be on point instead of long two-line sentences. Just compare the previous vs the new version:
* Example for a basic example using the transfer learning capabilities
* Transfer Learning

--> We can make the latter also more specific, especially when we add more TL examples. Something like "Leveraging existing data" or similar already conveys the key message

And for the specific one you asked about, let me finally use the opportunity to answer in the style of [this blog](https://jml.io/test-docstrings/) (was waiting for the right moment since ages):

* `Necessary imports for this example` (original version in other files)
--> Ah, good to know, I thought these imports are for the example you show 5 pages later.

* `Necessary imports`
--> Thanks for pointing out. I assume you show the "unnecessary" imports below then?

Hence: "Imports"

@Scienfitz, what are your general thoughts on this? Agree/Disagree?

Reply:
Then we do not need any header at all. When the first line starts with `import`, then having the header `Imports` is redundant :P
Jokes aside, this makes sense.